src/index.css

```diff
@@ -281,6 +283,22 @@ h1 {
   cursor: pointer;
   transition: border-color 0.25s;
 }
+
+.articleByTopics {
+  display: inline-grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  max-width: 100%;
+  /* flex-direction:column; */
+  /* justify-content: space-between; */
```

Comment:
Please always remove comments before submitting. Doing this in a workplace would get your code rejected.